src/index.ts

```diff
@@ -0,0 +1,76 @@
+import "dotenv/config";
+import express from "express";
+import bodyParser from "body-parser";
+import { createClient } from "redis";
+
+import { scheduleJobs } from "./jobs.js";
+import {
+  syncServers,
+  syncHashes,
+  storeMessage,
+  searchByRecipient,
+  searchByHash,
+} from "./util.js";
+
+const app = express();
+app.use(bodyParser.json());
+
+// Ensure environment variables are set, otherwise throw an error
+if (!process.env.REDIS_URL) {
+  throw new Error("REDIS_URL environment variable is required");
+}
+if (!process.env.PORT) {
+  throw new Error("PORT environment variable is required");
+}
+
+export const redis = createClient({
+  url: process.env.REDIS_URL,
+});
+
+redis.on("error", (err) => console.error("Redis Client Error", err));
+await redis.connect();
+
+scheduleJobs();
+
+// TODO: more error handling and HTTP response codes
```

Comment:
Convert this to an issue instead of inline for better tracking and contributions plz!

Reply:
I didn't make that comment 💀